firmware/port.c

```diff
@@ -442,7 +442,7 @@ ExecStatus port_begin_cmd(PortData *p) {
             sercom_i2c_master_init(p->port->uart_i2c, p->arg[0]);
             pin_mux(p->port->sda);
             pin_mux(p->port->scl);
-            sercom(p->port->uart_i2c)->I2CM.INTENSET.reg = SERCOM_I2CM_INTENSET_SB | SERCOM_I2CM_INTENSET_MB;
+            sercom(p->port->uart_i2c)->I2CM.INTENSET.reg = SERCOM_I2CM_INTENSET_ERROR;
```

Comment:
I don't think it's any open bug, but I saw I2C transfers hanging when it tried to move to the next command on the wrong interrupt, when the I2C controller wasn't actually ready.